elastic-search-support/src/main/java/uk/gov/hmcts/ccd/definition/store/elastic/mapping/CaseMappingGenerator.java

```diff
@@ -13,11 +13,15 @@
 import uk.gov.hmcts.ccd.definition.store.elastic.mapping.type.TypeMappingGenerator;
 import uk.gov.hmcts.ccd.definition.store.repository.entity.CaseFieldEntity;
 import uk.gov.hmcts.ccd.definition.store.repository.entity.CaseTypeEntity;
+import uk.gov.hmcts.ccd.definition.store.repository.entity.SearchAliasFieldEntity;
 
 @Component
 @Slf4j
 public class CaseMappingGenerator extends MappingGenerator {
 
+    private static final String ALIAS_TEXT_SORT_SUFFIX = "_sort";
+    static final String ALIAS_CASE_FIELD_PATH_PLACE_HOLDER = "<caseFieldPathPlaceHolder>";
```

Comment:
private

Reply:
I was using the static field in CaseMappingGeneratorTest so kept it package-private.